docker/checkout.bash

```diff
@@ -107,3 +107,4 @@ cd V2X-Hub
 git config core.sparsecheckout true
 git sparse-checkout init
 git sparse-checkout set ext/ccserver
+sudo apt-get update && sudo apt-get install -y libdlib-dev  
```

Comment:
This can be removed after syncing with `develop` since the dlib installation is handled as part of https://github.com/usdot-fhwa-stol/carma-platform/blob/3707df6886e4fa7bf0e80dbcbc0356e0e2d78377/docker/install.sh#L37